src/double_pendulum_as_single_pendulum.py

```diff
@@ -30,7 +29,7 @@ def _DoCalcVectorOutput(self, context, double_pend_state, unused, torque):
         # Extract manipulator dynamics.
         q = double_pend_state[:2]
         v = double_pend_state[-2:]
-        (M,Cv,tauG,B) = manipulator_dynamics(self.tree,q,v)
+        (M,Cv,tauG,B) = ManipulatorDynamics(self.tree, q, v)
```

Comment:
good.  yes class name should be CamelCase typically in python world

Reply:
note: this is a method, not a class.  
we have a python style guide here:  http://drake.mit.edu/code_style_guide.html?highlight=python%20style#python-style

it recommends that normally method names should be manipulator_dynamics (which is what I had initially) but it also talks about matching backwards portability.  Since the other RigidBodyTree methods are camelCase or CamelCase (depending on how expensive they are to call), i've claimed that exception here.  but getting consensus on this would be good.

Reply:
Hm, it would seem preferable to have functions just follow one naming convention, rather than two or three?

Why not just switch everything to `lower_case_with_underscores` per PEP 8 style guide?

Is there discussion on this somewhere?

Reply:
I'll move this discussion to drakedev slack in #python channel

Reply:
The current goal that I have for Drake is to have everything that is pure Python be PEP8, and everything that is just wrapped maintain the C++ names (for relative ease of finding the original), so I think perhaps these functions should be PEP 8 per that; however, it does mean that users have to deal with multiple conventions.

If consistency for users / students is most important, then perhaps it should stay `CamelCase` or `mixedCase`; if we would like to have pure Python code stick to PEP 8, then it should be `snake_case`?
(If/when we make more Pythonic wrappings of `pydrake`, those will most likely be PEP 8.)